subsystem-arm/implementations/joints/joint-router.hpp

```diff
@@ -32,12 +32,30 @@ namespace sjsu::arm
             return arguments;
         }
 
-        void HomeArm(arm_accelerometer_feedback mpu_feedback, motors_feedback motors_feedback)
+       // void HomeArm(arm_accelerometer_feedback mpu_feedback, motors_feedback motors_feedback)
+          void HomeArm() 
         {
-            
+            initial_rotunda_position_  = MapEncoderDataToDegrees(rotunda_.RequestFeedbackFromMotor().GetFeedback().encoder_position);
+            initial_shoulder_position_  = MapEncoderDataToDegrees(shoulder_.RequestFeedbackFromMotor().GetFeedback().encoder_position);
+            initial_elbow_position_  = MapEncoderDataToDegrees(elbow_.RequestFeedbackFromMotor().GetFeedback().encoder_position);
+            initial_left_wrist_position_  = MapEncoderDataToDegrees(left_wrist_.RequestFeedbackFromMotor().GetFeedback().encoder_position);
+            initial_right_wrist_position_  = MapEncoderDataToDegrees(right_wrist_.RequestFeedbackFromMotor().GetFeedback().encoder_position);
```

Comment:
```suggestion
            initial_rotunda_position_  = MapEncoderDataToDegrees(rotunda_.RequestFeedbackFromMotor().GetFeedback().encoder_position >> 8);
            initial_shoulder_position_  = MapEncoderDataToDegrees(shoulder_.RequestFeedbackFromMotor().GetFeedback().encoder_position >> 8);
            initial_elbow_position_  = MapEncoderDataToDegrees(elbow_.RequestFeedbackFromMotor().GetFeedback().encoder_position >> 8);
            initial_left_wrist_position_  = MapEncoderDataToDegrees(left_wrist_.RequestFeedbackFromMotor().GetFeedback().encoder_position >> 8);
            initial_right_wrist_position_  = MapEncoderDataToDegrees(right_wrist_.RequestFeedbackFromMotor().GetFeedback().encoder_position >> 8);
```
Corey said shift right by 8 is needed